wGetStarted/L-Dev-SimpleDataSave.md

```diff
@@ -484,40 +510,46 @@ This section will take the XML data sent by Interject from your report and put i
 
 ### Validations
 
-Now that you have all the data from your report into a table, you can do some validations on it before your target data source is updated. This section will run through certain validations and marks the \_MessageToUser upon a check. This is example code. You can use and modify it to match your report and data or you can delete it. There are 5 parts:
+Now that you have all the data from your report into a table, you can do some validations on it before your target data source is updated. This section will run through certain validations and marks the \_MessageToUser parameter upon a check. This is example code. You can use and modify it to match your report and data or you can delete it. There are 4 validation checks in the example code:
 
 1. Validating a parameter is a certain value (not used in this example)
 1. Validating the data to save does not contain duplicate keys (change data type and size to match your data)
 1. Validating a required column is not blank (not used in this example)
 1. Validating a column has certain text (not used in this example)
-1. If any validation checks recorded a message, this SP will GOTO the end and not update any records (it will show the user the message on the corresponding row)
+
+If any validation checks recorded a message, this SP will GOTO the end and not update any records. It will also record the message on the same row where the validation check encountered an error. Thus it can display this message back to the user on the same Excel row.
 
 ![](/images/L-Dev-SimpleDataSave/SPValidation.png)
 <br>
 
-### ChangeLog
+### Example Table To Update
 
 You can delete the "CREATE TABLE #ExampleTableToUpdate" part, it is not used in this example. 
 
+![](/images/L-Dev-SimpleDataSave/ExampleTableToUpdate.png)
+<br>
+
+### Change Log
+
 Next, another table variable is declared. The "@ChangeLog" table will be filled with information regarding the changes to the data, such as the row in the Excel report, the type of change that was done, and the corresponding key.
 
 ![](/images/L-Dev-SimpleDataSave/SPDeclareChangeLog.png)
 <br>
 
 ### Merge
 
-This section is where the data from your report (located in the "@DataToProcess" table), is merged with the target table in your database. It first declares a transaction. This is so the following changes can either be committed if no errors or rolled back if there is an error. 
+The Merge statement essentially merges two tables together based on certain criteria. In this example, you are merging the @DataToProcess table which comes from your Excel report with the target table in the database. The Merge statement can add data that is present in one table into the other table. It can also update data that has been changed by comparing the two.
```

Comment:
might be useful to explain here how `MERGE` is different from `JOIN` since both "merge" data from two tables, but `JOIN` is a more standard SQL approach. `MERGE` can modify data in a table by combining data from another. this is a key difference with `JOIN` and why its useful for Interject Saves in my opinion

Reply:
I added an explanation.